src/Chapter2.hs

```diff
@@ -349,7 +371,11 @@ Implement a function that returns only the first half of a given list.
 "b"
 -}
 -- PUT THE FUNCTION TYPE IN HERE
-firstHalf l = error "firstHalf: Not implemented!"
+firstHalf :: [a] -> [a]
+firstHalf l = subList 0 endIdx l
```

Comment:
Nice way of using `subList`! Though, in this case it's enough to _take_ only the first half of the list :slightly_smiling_face: